app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+from datetime import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date(datetime.today()):
+            raise OutdatedVaccineError
+        if not visitor["wearing_a_mask"]:
```

Comment:
In case the developer is sure of the structure of the incoming `visitor` dict, it is safe to leave the value retrieval for the `wearing_a_mask` key as is. In the production code in most cases it is better to check for the presence of a key before retrieving a value, that is why you could consider the safer method of retrieving a value and checking for the presence of the key at the same time.